### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Nine-JuanJuan**
Converted to DVM3.5 templates. Added options in inventory.example for updates. Will need to review

### CODE REVIEW

**Comment by Nine-JuanJuan** [comment]
@lastation4 Review changes before merging

File: playbooks/ctrs-installer.yml
Comment (faultywarrior) [inline playbooks/ctrs-installer.yml:1]: amd64 fix is half-correct.  We still need to do the ansible search for x86_64.

**Review by faultywarrior — APPROVED** [review]
lgtm

File: playbooks/ctrs-installer.yml [hunk 1]
```diff
@@ -74,40 +86,56 @@
...
-        dest: /opt/centrunk/dvm/dvmhost
+        src: /opt/centrunk/util/bin/amd64/dvmhost
+        dest: /opt/centrunk/dvmhost
         owner: root
         group: root
         mode: '0744'
         remote_src: yes
-      when: ansible_architecture is search("x86_64")
-    - name: Move in dvmcmd for x86_64
+      when: ansible_architecture is search("amd64")
```
Comment (lastation4) [inline playbooks/ctrs-installer.yml:125]: This needs to be reverted back to

"x86_64"

File: playbooks/ctrs-installer.yml [hunk 2]
```diff
@@ -74,40 +86,56 @@
...
-        src: /opt/centrunk/util/bin/x86_64/dvmcmd
-        dest: /opt/centrunk/dvm/dvmcmd
+        src: /opt/centrunk/util/bin/amd64/dvmcmd
+        dest: /opt/centrunk/dvmcmd
         owner: root
         group: root
         mode: '0744'
         remote_src: yes
-      when: ansible_architecture is search("x86_64")
+      when: ansible_architecture is search("amd64")
```
Comment (lastation4) [inline playbooks/ctrs-installer.yml:134]: This needs to be reverted back to

"x86_64"

**Review by lastation4 — CHANGES_REQUESTED** [review]
changes need to be made, I'll make them and re-commit

**Pull request closed without merging**